dev/windows/setup.ps1

```diff
@@ -0,0 +1,141 @@
+param (
+  [Alias("PSPath")]
+  [string] $phpDir
+)
+
+# check if the script param is empty
+if ([string]::IsNullOrEmpty($phpDir)) {
+  $phpDir = Read-Host "Enter the path of your PHP installation (e.g. 'C:\Program Files\php', 'D:\php', 'C:\wamp\bin\php')"
+}
+
+$fullPhpDir = Join-Path $phpDir "php.exe"
+
+# check if path exists
+if (-Not (Test-Path $phpDir)) {
+  throw "The directory '$phpDir' does not exist. Did you point to the exact file instead of the containing folder?"
+}
+
+# check if php.exe is there
+if (-Not (Test-Path $fullPhpDir)) {
+  throw "'php.exe' was not found in '$phpDir'. Was it installed correctly?"
+}
+
+Write-Host "Using $fullPhpDir as PHP runner."
+
+
+function Validate-Ini {
+  $confirm = Read-Host "Would you like for the script to validate your PHP`'s .ini file? [y/n]"
```

Comment:
The presence and correctness of `php.ini` should unconditionally be checked, since setup can't proceed without it. If issues are found, *then* ask the user if they would like the script to automatically fix them.

If the location of `php` is provided or found automatically, and there are no problems with `php.ini`, this script should prompt the user zero times.